composer.json

```diff
@@ -12,8 +12,9 @@
     }
   ],
   "require": {
-    "php": "~8.1.0 || ~8.2.0",
-    "pimcore/pimcore": "^10.5 || ^11.0"
+    "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+    "doctrine/doctrine-fixtures-bundle": "^3.4",
+    "pimcore/pimcore": "^10.5 || ^11.2"
```

Comment:
Is there a reason you're not allowing Pimcore `11.0` & `11.1`?

Reply:
I don't want to support older versions and make my work harder than it already is.